software/o_c_REV/APP_HEMISPHERE.ino

```diff
@@ -48,20 +49,23 @@ typedef struct Applet {
   void (*OnButtonPress)(bool); // Encoder button has been pressed
   void (*OnEncoderMove)(bool, int); // Encoder has been rotated
   void (*ToggleHelpScreen)(bool); // Help Screen has been requested
-  uint32_t (*OnDataRequest)(bool); // Get a data int from the applet
-  void (*OnDataReceive)(bool, uint32_t); // Send a data int to the applet
+  uint64_t (*OnDataRequest)(bool); // Get a data int from the applet
+  void (*OnDataReceive)(bool, uint64_t); // Send a data int to the applet
 } Applet;
 
 // The settings specify the selected applets, and 32 bits of data for each applet
 enum HEMISPHERE_SETTINGS {
     HEMISPHERE_SELECTED_LEFT_ID,
     HEMISPHERE_SELECTED_RIGHT_ID,
-    HEMISPHERE_LEFT_DATA_L,
-    HEMISPHERE_RIGHT_DATA_L,
-    HEMISPHERE_LEFT_DATA_H,
-    HEMISPHERE_RIGHT_DATA_H,
-    HEMISPHERE_CLOCK_DATA,
-    HEMISPHERE_SETTING_LAST
```

Comment:
Yeah I was having build problems that were unrelated and in attempting to fix them I moved this out into a define (as it is in NeuralNet and Enigma, which don't have the APPNAME_SETTINGS enums) because I thought it was supposed to be the number of bytes used, not number of settings. I'll be moving it back assuming it works.